libraries/AP_Motors/AP_MotorsMatrix.h

```diff
@@ -127,6 +144,10 @@ class AP_MotorsMatrix : public AP_MotorsMulticopter {
 
     const char*         _frame_class_string = ""; // string representation of frame class
     const char*         _frame_type_string = "";  //  string representation of frame type
+
 private:
     static AP_MotorsMatrix *_singleton;
+
+    void add_motors(const struct MotorDefInt *motor, uint8_t num_motors);
```

Comment:
Let's put a comment above this new method explaining why it exists.  Also putting this discussion just below the other existing add_motors() would be nice.

Reply:
Added it.  Especially in context with the structure I think it's straight forward.